test/zim.test.ts

```diff
@@ -218,6 +217,18 @@ describe("Archive", () => {
       )
   );
 
+  // test blobs via StringItem
+  const blobs = Array.from(Array(3).keys()).map(
+    (i) =>
+      new StringItem(
+        `blob${i}`,
+        "application/octet-stream",
+        `blob title ${i}`,
+        {},
+        crypto.createHash("md5").update(`blob title ${i}`).digest()
```

Comment:
It may be interesting to be sure there is a `\0` in the data to check we create the stream correctly.

Reply:
Added a test for it.